radio/src/gui/9X/menu_model_telemetry.cpp

```diff
@@ -606,7 +606,7 @@ void menuModelTelemetry(uint8_t event)
     switch (k) {
 #if defined(CPUARM)
       case ITEM_TELEMETRY_PROTOCOL_TYPE:
-        g_model.telemetryProtocol = selectMenuItem(TELEM_COL2, y, STR_TELEMETRY_TYPE, "\017FrSky S.PORT\0  FrSky D\0       FrSky D (cable)", g_model.telemetryProtocol, PROTOCOL_TELEMETRY_FIRST, CASE_PCBSKY9X(PROTOCOL_FRSKY_D_SECONDARY) attr, event);
+        g_model.telemetryProtocol = selectMenuItem(TELEM_COL2, y, STR_TELEMETRY_TYPE, "\017FrSky S.PORT\0  FrSky D\0       FrSky D (inv)\0 FrSky D (cable)", g_model.telemetryProtocol, PROTOCOL_TELEMETRY_FIRST, CASE_PCBSKY9X(PROTOCOL_FRSKY_D_SECONDARY) attr, event);
```

Comment:
The problem with not putting it as last item is that anyone who had his mode configured as FrSky D (cable) will see his choice now being interpreted as FrSky D (inverted), his setup won't work anymore, and he won't understand why. 
If you want to be clean and group the settings then your should target your PR for the next major version (2.2), base your branch on `next`, and implement an eeprom conversion routine that handles the shift when upgrading the eeprom format.
Note that if you change it as last item you of course have to move the new item in myeeprom.h too.